android/build.gradle

```diff
@@ -20,6 +20,7 @@ def _minSdkVersion = _ext.has('minSdkVersion') ? _ext.minSdkVersion : 16
 def _targetSdkVersion = _ext.has('targetSdkVersion') ? _ext.targetSdkVersion : 27
 
 android {
+    namespace = "com.corbt.keepawake"
```

Comment:
https://github.com/react-native-community/discussions-and-proposals/issues/671#issuecomment-1598892444

You need to wrap it in this code to maintain backward compatibility